### PR TITLE
Add Name Clash Info To Docs

### PR DESCRIPTION
**Pull request by dickermoshe**

### CODE REVIEW

File: docs/pages/docs/Dart API/manager.md [hunk 1]
```diff
@@ -60,6 +60,12 @@ We can now use them in a query like this:
 In this example, had we not specified a custom name for the reference, the code generator would have named both filtersets `userRefs` for both references to the `User` table. This would have caused a conflict. By specifying a custom name, we can avoid this issue.
 
 
+### Name Clashes
```
Comment (simolus3) [inline docs/pages/docs/Dart API/manager.md]: Imo this could be a `####` heading, then it shows neatly under the code snippet using the annotation.

Comment (simolus3) [reply docs/pages/docs/Dart API/manager.md]: Giving this an anchor also gives us a opportunity to reference drift.simonbinder.eu/docs/manager/#name-clashes in the error message.

**Review by simolus3 — COMMENTED** [review]
That was quick, thanks!!

I think we should also reference the updated docs in the error message to perhaps provide more context.

File: docs/pages/docs/Dart API/manager.md (hunk 1 above)
Comment (dickermoshe) [reply docs/pages/docs/Dart API/manager.md]: Done :)

**Review by simolus3 — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**